SKILL.md

```
@@ -41,6 +41,81 @@ Thin session manager for Loki Mode. Classify work, spec it via OpenSpec CLI, con
+
+```bash
+bash <skill-path>/scripts/drain.sh                          # drain all open issues
+bash <skill-path>/scripts/drain.sh --label bug              # drain only bugs
+bash <skill-path>/scripts/drain.sh --label refactor --max-iterations 3
+```
+
+Or via the skill trigger: `wishloop --drain` / `wishloop --drain --label bug`
+
+### How It Works
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)